### PR TITLE
Use non-interactive backend in nwl utils to avoid matplotlib cubit conflict

### PR DESCRIPTION
**Pull request by Edgar-21**
With the newest version of openmc (0.15.0) the previous conflict with Cubit seems to have been resolved. The remaining conflict with matplotlib can be resolved as in this PR, allowing the full NWL workflow to be performed in one script. In a future PR the nwl_utils example could be updated to be a single script.

### CODE REVIEW

**Comment by connoramoreno** [comment]
We should perhaps consider including OpenMC v0.15.0 in the environment YAML file

File: parastell/nwl_utils.py [hunk 1]
```diff
@@ -5,8 +5,10 @@
 import pystell.read_vmec as read_vmec
 import matplotlib.pyplot as plt
 import concurrent.futures
-from multiprocessing import cpu_count
```
Comment (connoramoreno) [inline parastell/nwl_utils.py:8]: Why are we removing `multiprocessing`? Wouldn't that remove parallelization from the NWL routine?

**Review by connoramoreno — COMMENTED** [review]
Just a small question about removing the `multiprocessing` module from the list of imports.

File: parastell/nwl_utils.py (hunk 1 above)
Comment (Edgar-21) [reply parastell/nwl_utils.py:8]: `concurrent.futures` is what does the multiprocesssing, I originally attempted to make some guess about how many cpus to use based on `cpu_count` but ended up removing that bit, but this import remained

**Comment by connoramoreno** [comment]
I'm in favor of at least including some mention that OpenMC v0.15.0 is needed to use the NWL utility. I'm not opposed to including it as a hard dependency to prevent user error on that front, although I recognize it complicates the environment and may become a problem in the future, like it was in the past.

**Comment by connoramoreno** [comment]
Looks good, thanks @Edgar-21

**Pull request merged**